public/app/features/users/TokenRevokedModal.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { Button, InfoBox, Portal, stylesFactory, useTheme } from '@grafana/ui';
+import { getModalStyles } from '@grafana/ui/src/components/Modal/getModalStyles';
+import { css, cx } from 'emotion';
+
+const getStyles = stylesFactory(() => {
+  return {
+    infobox: css`
+      margin-bottom: 0;
+    `,
+    text: css`
+      margin: 8px 0 16px;
+    `,
+    backdrop: css`
+      background-color: #0b0d10;
```

Comment:
Can we use a variable here instead of hardcoding a color?

Reply:
Sure, `theme.colors.dashboardBg` looks like a good candidate! Thanks for the recommendation! 🙏🏻